SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1,4 +1,5 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma optimize("", off)
```

Comment:
left over optimize off

Reply:
side note epic provides the slightly easier to auto complete: `PRAGMA_DISABLE_OPTIMIZATION PRAGMA_ENABLE_OPTIMIZATION`

Reply:
nice tip, ty